src/createServer.js

```diff
@@ -1,3 +1,46 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+'use strict';
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase.js');
+const { messageError } = require('./services/messageError.js');
+
+const createServer = () => {
+  const server = http.createServer((request, respons) => {
+    respons.setHeader('Content-type', 'application/json');
+
+    const [inputText, inputCase] = request.url.split('?');
+    const originalText = inputText.slice(1);
+    const targetCase = new URLSearchParams(inputCase).get('toCase');
+    const errors = messageError(originalText, targetCase);
+
+    const {
+      originalCase,
+      convertedText,
+    } = convertToCase(originalText, targetCase);
```

Comment:
you see errors, because u don't validate `targetCase`, so its nullish/invalid value goes to this function, so an error should be thrown before the execution of `convertToCase`

consider creating some validations for targetCase value before this row